tests/user.test.ts

```diff
@@ -184,4 +184,27 @@ import { Marqeta } from '../src'
     console.log(listA)
   }
 
+  console.log('testing list one User transitions by token Id...')
+  if (testUser?.token) {
+    let listTransOne
+    listTransOne = await client.user.listTransition({
+      ...testUser,
+      count: 1,
+    })
+    if (listTransOne?.success
+      && listTransOne?.body?.count && listTransOne.body) {
+      console.log(`[ONE: ] ${JSON.stringify(listTransOne)}`)
+      console.log('Success! We were able to get a list of one User ' +
+        'transitions.')
```

Comment:
Why not make this one, longer, line? Yeah, it'll exceed 80 cols, but it's a test, and it's not by that much...